sass/pages/_scheduler.scss

```diff
@@ -0,0 +1,57 @@
+
+.scheduler-grid {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+    background: white;
+
+    .scheduler-row {
+        display: table-row;
+
+        .row-info, .row-grid {
+            display: table-cell;
+            border-bottom: 1px solid #f0f0f0;
+            min-height: 46px;
+        }
+
+        .row-info {
+            width: 180px;
+            vertical-align: middle;
+            padding-left: 10px;
+        }
+
+        .week-row {
+            display: table;
+            empty-cells: show;
+            table-layout: fixed;
+            width: 100%;
+            min-height: 40px;
+        }
+        .day-cell {
+            display: table-cell;
+            width: 14.28%;
+            border-left: 1px solid #f0f0f0;
+
+            .shift-holder {
+                padding: 5px 5px 0;
+            }
+        }
+
+        .shift-item {
+            display: block;
+            padding: 5px;
+            margin-bottom: 5px;
+            p {
+                margin: 0;
+                color: white;
+                font-size: 14px;
+                line-height: 26px;
+
+                strong {
```

Comment:
This is a bad example, but there is definitely cases where you'll want different levels of bold-ness. :smile: